src/bin/dev.ts

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+import config from "../config";
+import {ScriptLinter, TestRunner} from "../scripts";
+import {StyleCompiler, StyleLinter} from "../styles";
+import {TranslationCompiler} from "../translations";
+import * as del from "node-delete";
+import "../hmr";
+import "../livereload";
+
+del.sync(".tmp");
+
+new ScriptLinter(
+  config.scripts.lint.files,
+  config.scripts.lint.exclude,
+  true)
+  .start();
+
+new StyleLinter(
+  config.styles.lint.files,
+  true)
+  .start();
+
+new StyleCompiler(
+  config.styles.entry,
+  config.styles.dev.output,
+  false,
+  config.styles.dev.watch)
+  .start();
+
+new TranslationCompiler(
+  config.translations.files,
+  config.translations.output,
+  true,
+  false)
+  .start();
+
+new TestRunner(true).start();
```

Comment:
Hm... I'm not sure if I like this. Several of the parameters used here are hard-coded (but not part of the config), without any possibility to overwrite them via CLI. This would force anyone to either accept these as fixed parameters, or use the source tasks manually. I'm fine with using these as default parameters, but it should be possible to provide them in a different way.